service/cmd/server/record.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/livekit/protocol/utils"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/livekit/livekit-recorder/service/pkg/config"
+	"github.com/livekit/livekit-recorder/service/pkg/logger"
+	"github.com/livekit/livekit-recorder/service/pkg/service"
+	livekit "github.com/livekit/livekit-recorder/service/proto"
+)
+
+// Acts as a livekit server against a recorder-service docker container with shared redis
```

Comment:
who uses these functions? are these for testing?

Reply:
yeah, they're mostly for testing. But could be used for one-off recordings